src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -174,9 +174,9 @@ function checkPrivilegesOfRoleForInstructor(instrNum, role) {
 		checkPrivilegesOfTutorForInstructor(instrNum);
 	} else if (role === "Custom") {
 		checkPrivilegesOfCustomForInstructor(instrNum);
-	} else {
-
 	}
+    
+	// do nothing if role not recognized
```

Comment:
I like this better, the previous implementation left no clue to maintainers as to why it's empty :+1:, although it'll be even better if we can find a way to not let it silently fail but I still think this is an improvement :)